Formula/ccfits

```diff
@@ -0,0 +1,23 @@
+class Ccfits < Formula
+  desc "c++ fits api"
+  homepage "https://heasarc.gsfc.nasa.gov/fitsio/CCfits/"
+  url "https://heasarc.gsfc.nasa.gov/fitsio/CCfits/CCfits-2.5.tar.gz"
+  sha256 "938ecd25239e65f519b8d2b50702416edc723de5f0a5387cceea8c4004a44740"
+
+  bottle do
+    cellar :any
+    sha256 "504d1af8fda5926434fc3045b1426e50f9e916bb764058773d1ef95c7084c43a" => :catalina
+  end
+
+  depends_on "cfitsio"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make"
+    system "make", "check" if build.with? "check"
+    system "make", "install"
+  end
+end
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
delete checked

Reply:
Please look at other formula and create a `test` block.

Reply:
test block added

Reply:
What part of the software does this test?

Reply:
the program has its test program as cookbook

Reply:
That's not what this tests though. you'd need something like `assert_match "some output", shell_output("#{bin}/cookbook")` for that. The current test only checks if you can make a file called `test.sh`. What might be even better than running the test program though is running something the user would run in general usage of the app. Since the user's interaction is what we want to test after all.

Reply:
sorry, I rewrite the test block. and tested